src/helpers.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace LaravelDoctrine\Fluent;
+
+use Doctrine\ORM\Mapping\MappingException;
+use FilesystemIterator;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use RecursiveRegexIterator;
+use ReflectionClass;
+use RegexIterator;
+
+/**
+ * Returns an array of Mapping objects found on the given paths.
+ *
+ * @param string[] $paths
+ * @param string   $fileExtension
+ *
+ * @throws MappingException
+ *
+ * @return Mapping[]
+ */
+function mappingsFrom(array $paths, $fileExtension = '.php')
+{
+    $includedFiles = [];
+
+    foreach ($paths as $path) {
```

Comment:
@patrickbrouwers I'm pretty sure we should be able to replace this loop with another iterator, so that we iterate recursively over each path, instead of building the full iterator chain for each path in the array.